.github/workflows/CI.yml

```diff
@@ -375,12 +375,16 @@ jobs:
 
       - run: ls -R
 
+        # Build the ssm package and manifest by using a version with github short sha as same as e2etest-preparationfor the following reasons:
```

Comment:
```suggestion
        # Build the ssm package and manifest by using a version with github short sha same as e2etest-preparation for the following reasons:
```

Reply:
Reasonable. Fixed it in the newest commit.